website/content/docs/connect/ca/vault.mdx

```diff
@@ -279,6 +317,137 @@ path "/connect_inter/*" {
+multiple, isolated Vault environments within a single Vault infrastructure.
+
+This Connect CA provider can access Vault tokens, auth methods, and PKI secrets engines
+that exist within the same or different Vault Namespaces.
+
+The [`Namespace`](#namespace) option is used as the `X-Vault-Namespace` header
+for all API requests made by this CA provider to Vault.
+All Vault namespaces included in paths are relative to the value of this header.
+For more details on how paths to namespaced resources can be specified,
+refer to the [Vault Namespace usage documentation](https://www.vaultproject.io/docs/enterprise/namespaces#usage).
+
+The PKI path options ([`RootPKIPath`](#rootpkipath) and [`IntermediatePKIPath`](#intermediatepkipath)
+both support including a Vault Namespace in the path.
+However, the Vault authentication options ([`Token`](#token) and [`AuthMethod`](#authmethod))
+do not allow specifying a Vault Namespace in their configuration.
+
+**If using an authentication mechanism that exists within a non-root Vault Namespace**:
+- set the [`Namespace`](#namespace) option to the Vault authentication mechanism's Namespace, and
+- `RootPKIPath` and `IntermediatePath` must be in that same Namespace or a child Namespace;
+  the relative path to a child Namespace must be included in the path
+
+### Example 1: PKI Paths in Same Namespace
+
+Consider the following setup of Vault resources and their namespaces:
+
+| Vault Resource                       | Belongs to Vault Namespace | Resource Name              |
+| ------------------------------------ | -------------------------- | -------------------------- |
+| `Token`                              | root                       | N/A                        |
+| `RootPKIPath` (Primary CA)           | `ns-consul`                | `connect-root`             |
+| `IntermediatePKIPath` (Secondary CA) | `ns-consul`                | `connect-intermediate-dc1` |
+
+The correct Connect CA configuration for this setup is:
+
+<CodeTabs heading="Connect CA configuration" tabs={["Agent configuration", "API"]}>
+
+<CodeBlockConfig filename="/etc/consul.d/config.hcl" highlight="7-9">
+
+```hcl
+# ...
+connect {
+  enabled = true
+  ca_provider = "vault"
+  ca_config {
+    address = "http://localhost:8200"
+    token = "<Vault token>"
+    root_pki_path = "ns-consul/connect-root"
+    intermediate_pki_path = "ns-consul/connect-intermediate-dc1"
+  }
+}
+```
+
+</CodeBlockConfig>
+
+<CodeBlockConfig highlight="5-7">
+
+```json
+{
+  "Provider": "vault",
+  "Config": {
+    "Address": "http://localhost:8200",
+    "Token": "<Vault token>",
+    "RootPKIPath": "ns-consul/connect-root",
+    "IntermediatePKIPath": "ns-consul/connect-intermediate-dc1"
+  }
+}
+```
+
+</CodeBlockConfig>
+
+</CodeTabs>
+
+-> **Why the Namespace option isn't used**:
+  In this case, the [`Namespace`](#namespace) option cannot be set to `ns-consul/` because
+  the Vault `Token` is in the root Vault Namespace.
```

Comment:
```suggestion
The [`Namespace`](#namespace) option is not included in this case because the Vault `Token` is in the root Vault namespace. As a result, the Consul `Namespace` cannot be set to `ns-consul/`. 
```
I think this is directly related and probably doesn't need a callout box